src/applications/vre/28-1900/orientation/OrientationApp.jsx

```diff
@@ -17,18 +15,21 @@ const scrollToTop = () => {
 
 const OrientationApp = props => {
   const [step, setStep] = useState(0);
-  const [formStartControl, setFormStartControl] = useState(false);
+  const [isFirstRender, setIsFirstRender] = useState(true);
   const { wizardStateHandler } = props;
 
   // do this to prevent focus skip on prior link tag
-  useEffect(() => {
-    if (formStartControl) {
-      focusElement('#FormStartControl');
-    } else {
-      focusElement('#StepTitle');
-      scrollToTop();
-    }
-  });
+  useEffect(
+    () => {
+      if (isFirstRender && step === 0) {
```

Comment:
This is done to prevent focus skip of the link rendered at the end of the wizard.